zoo/policies/cross-rl-agent/setup.py

```diff
@@ -0,0 +1,13 @@
+from cross_rl_agent.version import VERSION
+from setuptools import setup
+
+setup(
+    name="cross-rl-agent",
+    description="cross rl agent example",
+    version=VERSION,
+    packages=[
+        "cross_rl_agent",
+    ],
+    include_package_data=True,
+    install_requires=["tensorflow==1.15", "smarts"],
```

Comment:
We will want to snap the agent to a version of SMARTS likely. The other option is that we write a test to check zoo agent compatibility with SMARTS.
```suggestion
    install_requires=["tensorflow==1.15", "smarts==0.4.15"],
```

Reply:
On second thought, requiring SMARTS 0.4.15 will cause problems when installing the agent.